lesson07/assignment/restaurant/src/components/FoodMenu.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+ <!-- Fills out and binds attributes from the object array to FoodMenuItem components -->
+            <main class="main__primary-container col-sm-12 col-md-9 col-xl-8">
+            <h1>Food</h1> 
+              <section class="main__menu-list"> 
+                <FoodMenuItem 
+                v-for="item in items" 
+                v-bind:key="item.key"
+                v-bind:dishName="item.dishName"
+                v-bind:text="item.text"
+                v-bind:imageSrc="item.imageSrc"
+                />
+              </section>
+          </main>
+</template>
+
+<script>
+import FoodMenuItem from '@/components/FoodMenuItem.vue'
+
+export default {
+  name: 'FoodMenu',
+  components: {
+    FoodMenuItem
+  },
+  props: {},
+  data() {
+    return { items:
+      [
+        { id: 1, dishName:"Sushi", imageSrc:"sushi-1634013_960_720.jpg", text:"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua." },
```

Comment:
This order page is showing errors in the web console. This is because both the food items and drink items have the same key. Dynamically created elements must have unique IDs. Would recommend changing your IDs to something like "food-1", "food-2", etc.

Reply:
Ah, that makes sense.  I saw that error but wasn't sure if that was the right way to correct that, by changing the keys to be more unique to their object vs keeping them as numbers.